### PR TITLE
add lombok cheat sheet

### PR DESCRIPTION
**Pull request by marti1125**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [X] New Instant Answer
    - [X] Cheat Sheets: **`New Lombok Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**

Developer and Maintainer: @marti1125

---
https://duck.co/ia/view/lombok_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/lombok.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "name": "Project Lombok",
+    "description": "Makes java a spicier language by adding 'handlers' that know how to build and compile simple, boilerplate-free, not-quite-java code.",
+
+    "metadata": {
+        "sourceName": "Project Lombok",
+        "sourceUrl" : "https://projectlombok.org/features/index.html"
+    },
+
+    "aliases": [
+        "first alias", "project lombok"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/lombok.json]: Please could you remove `"first alias"`

File: share/goodie/cheat_sheets/json/lombok.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{
+    "id": "lombok_cheat_sheet",
+    "name": "Project Lombok",
+    "description": "Makes java a spicier language by adding 'handlers' that know how to build and compile simple, boilerplate-free, not-quite-java code.",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/lombok.json]: Please could java be uppercase, i.e. `Java`

File: share/goodie/cheat_sheets/json/lombok.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                "key": "@Builder",
+                "val": "No-hassle fancy-pants APIs for object creation!"
+            },
+            {
+                "key": "@SneakyThrows",
+                "val": "To boldly throw checked exceptions where no one has thrown them before!"
+            },
+            {
+                "key": "@Synchronized",
+                "val": "synchronized done right: Don't expose your locks."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/lombok.json]: Please could you make the first "s" uppercase, to be consistent with the other entries? I.e. `Synchronized`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=lombok cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/lombok_cheat_sheet_mobile.png?AASIBZehAP)

**Comment by tagawa** [comment]
Thanks for this @marti1125. It's now on the beta server for testing: https://beta.duckduckgo.com/?q=lombok+cheat+sheet&ia=cheatsheet

I made a couple of review comments in the code. Also, the ID in the JSON file has to be the same as the ID in the IA page, so I updated the IA page. This is needed for the automatic beta server installation.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=lombok cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/lombok_cheat_sheet_mobile.png?IGWLzULWyC)

File: share/goodie/cheat_sheets/json/lombok.json (hunk 1 above)
Comment (marti1125) [inline share/goodie/cheat_sheets/json/lombok.json]: done

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=lombok cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/lombok_cheat_sheet_mobile.png?01I86Qaaru)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=lombok cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/lombok_cheat_sheet_mobile.png?Qr3u2CZYet)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=lombok cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/lombok_cheat_sheet_mobile.png?VyJmArKSmy)

**Comment by sahildua2305** [comment]
Looks good. Thanks @marti1125

**Comment by daxtheduck** [comment]
@marti1125 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by marti1125** [comment]
thanks! guys =D

**Pull request merged**